packages/config/config/devices/0x0438/1_kanal_bryter.json

```diff
@@ -2,8 +2,8 @@
 {
```

Comment:
The comment at the top does not fit the style guide. See the style guide at Comments. https://zwave-js.github.io/node-zwave-js/#/config-files/style-guide